packages/connectivity/src/scp-cf/identity-service.spec.ts

```diff
@@ -37,3 +38,41 @@ describe('shouldExchangeToken', () => {
     ).toBe(false);
   });
 });
+
+describe('exchangeToken', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should call jwtBearerToken with destination service', async () => {
```

Comment:
Not sure if these tests add much, since the wrapping function is essentially just a wrapper that sets "destination" as default, I leave it to you if you want to leave it in.